src/RSEF/object_creator/create_downloadedObj.py

```diff
@@ -192,12 +192,131 @@ def pdf_to_downloaded_obj(pdf,output_dir):
     arxiv = extract_arxivID(resp_jsn)
     return DownloadedObj(title=titL,doi=doi,arxiv=arxiv,file_name="",file_path=pdf)
 
+# Download papers from a json containing title, doi, primary location (PDF's URL)
+def json_to_downloaded_obj(json_data, output_dir):
+    if not os.path.exists(json_data):
+        print(f"Error: JSON file '{json_data}' does not exist.")
+        return None
+    try:
+        # Clear the content of the file downloaded_metadata.json
+        downloaded_metadata_path = os.path.join(output_dir, "downloaded_metadata.json")
+        if os.path.exists(downloaded_metadata_path):        
+            with open(downloaded_metadata_path, 'w') as file:
+                file.truncate(0)
+
+        with open(json_data, 'r', encoding='utf-8') as file:
+            json_data = file.read()
+            if not json_data:
+                print(f"Error: JSON file is empty.")
+                return None
+            json_data_list = json.loads(json_data)
+            output_path = output_dir + "/" + "downloaded_metadata.json"
+            # Download the papers
+            download_by_json(json_data_list, output_path)
+
+            return output_path
+    except Exception as e:
+        print(f"Error decoding JSON data': {e}")
+        return None
+
 def download_from_doi(doi,output_dir):
     return doi_to_downloadedJson(doi,output_dir)
 def download_from_doi_list(dois,output_dir):
     return dois_to_downloadedJson(dois,output_dir)
 def download_from_doi_txt(dois_txt,output_dir):
     return dois_to_downloadedJson(dois_txt, output_dir)
+
+# Download a paper from it's url
+def download_from_pdf_url(url, output_dir):
```

Comment:
Please add documentation for every new function following the format above.
Also, it should be "its documentation"

Reply:
Done in the lastest commit